packages/mobile/lib/supabase.ts

```diff
@@ -0,0 +1,40 @@
+import "react-native-url-polyfill/auto";
+import { AppState, Platform } from "react-native";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { createClient, processLock } from "@supabase/supabase-js";
```

Comment:
Great job Peter! But I'm unable to start the app because I'm receiving an error from line number 4.